src/tests/Home.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Home from '../pages/Home';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(<Home />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Please add integration tests as specified on the requirements of this project.

> There should be the usage of React Testing Library to simulate user interaction.